src/templates/server_status.html

```diff
@@ -28,11 +28,24 @@
 
 <div class="jumbotron text-center">
   <h1>Server for {{namespace}}/{{project}}/{{commit_sha}} status: {{status}}</h1>
-  <p>This page will automatically refresh and redirect you to the server when it is running.</p>
-</div>
+  {% if status == "not found" %}
```

Comment:
For the moment this is a good improvement, but moving forward, the trick with the previous state in a cookie should be replaced with some server-side state handling (use the existing redis pod?). From a client side, I am just interested in the state of the server, and I should get a response for it without assuming that it was the same client which has triggered a certain action beforehand.